sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPChecks.java

```diff
@@ -55,11 +54,11 @@ public PHPChecks addChecks(String repositoryKey, Iterable<Class> checkClass) {
     return this;
   }
 
-  public PHPChecks addCustomChecks(@Nullable PHPCustomRulesDefinition[] customRulesDefinitions) {
-    if (customRulesDefinitions != null) {
+  public PHPChecks addCustomChecks(@Nullable PHPCustomRuleRepository[] customRuleRepository) {
```

Comment:
@pynicolas `customRuleRepository` -> `customRuleRepositories`?

Reply:
Done in 832a63fd.